.gitignore

```diff
@@ -67,3 +67,4 @@ installation/versions-overrides.env
 
 # Changelog temporary directory
 .changelog/
+components/event-bus/cmd/event-publish-service/event-publish-service
```

Comment:
That's a strange place to add an exclusion for a specific component. Better use the local Makefile.

Reply:
This was the IDE, sorry!